### PR TITLE
Elacey/23.10 upgrade

### PR DESCRIPTION
**Pull request by evberrypi**
Upgrade K8s versions to 1.27 in AKS (GKE auto upgrades, current release channel is 1.27)

### CODE REVIEW

**Review by MaggieXJZhang — COMMENTED** [review]
The PR description was a little confusing "Upgrade K8s versions to 1.27 in AKS and EKS (GKE auto upgrades, current release channel is 1.27)" 
EKS is already on 1.27 since last release and you don't actually modify the EKS version, so maybe reword a little bit?

File: aks/variables.tf [hunk 1]
```diff
@@ -25,7 +25,7 @@ variable "cluster_name" {
 }
 
 variable "kubernetes_version" {
-  default     = "1.27.3"
+  default     = "1.27"
```
Comment (MaggieXJZhang) [inline aks/variables.tf:28]: Please update the AKS readme to reflect this change

Comment (evberrypi) [reply aks/variables.tf:28]: Fixed

**Review by MaggieXJZhang — APPROVED** [review]
